src/scripts/components/layer/wv.layer.list.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import LayerRadio from './wv.layer.radio.js';
+import { CellMeasurer, CellMeasurerCache, List } from 'react-virtualized';
+
+/*
+ * A react component, Builds a list of layers using the LayerRadio component
+ * @class LayerList
+ * @extends React.Component
+ */
+export default class LayerList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      layerFilter: props.config.layerOrder,
+      active: props.model.active,
+      infoExpanded: [],
+      width: props.width,
+      height: props.height
+    };
+    this._cache = new CellMeasurerCache({
+      fixedWidth: true,
+      minHeight: 23,
+      minWidth: this.state.width
+    });
+
+    this._rowRenderer = this._rowRenderer.bind(this);
```

Comment:
What do you think about binding `this` using arrow functions instead of using separate lines inside the constructor?

Where we use a function in a component's prop, we can use `(row)=>this._rowRenderer(row)` instead of `this._rowRenderer`.